src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMFileSystemTest.java

```diff
@@ -225,16 +225,21 @@ public void readFileFromDir() throws Exception {
                 is("c0e024f89969b976da165eecaa71e09dc60c3da1"));
         SCMFileSystem fs = SCMFileSystem.of(source, master, revision);
 
-        // On windows, if somebody has not configured Git correctly, the checkout may have "fixed" line endings
-        // So let's detect that and fix our expectations.
         String expected = "Some text\n";
-        try (InputStream is = getClass().getResourceAsStream("/raw/__files/body-fu-bar.txt-b4k4I.txt")) {
-            if (is != null) {
-                expected = IOUtils.toString(is, StandardCharsets.US_ASCII);
-            }
-        } catch (IOException e) {
-            // ignore
-        }
+        // In previous versions of github-api, GHContent.read() (called by contentAsString())
+        // would pull from the "raw" url of the GHContent instance.
+        // Thus on windows, if somebody did not configure Git correctly,
+        // the checkout may have "fixed" line endings that we needed to handle.
+        // The problem with the raw url data is that it can get out of sync when from the actual content.
+        // The GitHub API info stays sync'd and correct, so now GHContent.read() pulls from mime encoded data
+        // in the GHContent record itself. Keeping this for refence in case it changes again.
+//        try (InputStream inputStream = getClass().getResourceAsStream("/raw/__files/body-fu-bar.txt-b4k4I.txt")) {
+//            if (inputStream != null) {
+//                expected = IOUtils.toString(inputStream, StandardCharsets.US_ASCII);
+//            }
+//        } catch (IOException e) {
+//            // ignore
+//        }
```

Comment:
Thank you for the explanation here.

Reply:
1.106 published and available.  It is only not here yet because we're wait for a repository sync so we can build against it (we think).  Just waiting for it to show up in https://repo1.maven.org/maven2/org/kohsuke/github-api/

Later today likely.   1.106 mostly a dependency downgrade to allow the plugin to use Jenkins 2.150.3.  I'll make a PR for it after I merge this.